docs/API.md

```diff
@@ -1073,6 +1073,20 @@ Example: `GET trusty/juju-gui-3/meta/charm-config`
 }
 ```
 
+#### GET *id*/meta/terms
+
+The `meta/terms` path returns a list of terms and angreements the user must
+agree to in order to obtain the archive of the given charm or bundle id.
```

Comment:
In fact, bundles can't have terms, so "bundle id" is wrong here AFAICS. In fact, is it actually worth having a separate terms endpoint when we can just fetch the charm metadata and get the terms from that?